chcemvediet/apps/inforequests/tests/__init__.py

```diff
@@ -173,7 +173,7 @@ def _create_message(self, **kwargs):
             u'received_for': u'default_testing_for_mail@example.com',
```

Comment:
@viliambalaz Pls daj hore do popisu PR linku na komentar z tasku, ktory tento PR implementuje. A tiez naopak zedituj dany komentar a daj tam linku na tento PR. Nech to je prelinkovane. Rovnako pri dalsich PR, co implementuju nieco, co je iba spomenute niekde v komentari.

*) Tento komentar nie je na konkretny riadok. Vesam ho sem len preto, aby sa dal thread potom resolvnut.

Reply:
Inac je PR OK. Prelinkuj to a PR approvnem.